blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineUpdateRequest.java

```diff
@@ -1,107 +0,0 @@
-package io.jenkins.blueocean.blueocean_github_pipeline;
-
-import hudson.model.Cause;
-import hudson.model.Item;
-import hudson.model.User;
-import hudson.security.ACL;
-import io.jenkins.blueocean.commons.ServiceException;
-import io.jenkins.blueocean.rest.model.BluePipeline;
-import io.jenkins.blueocean.rest.model.BluePipelineUpdateRequest;
-import io.jenkins.blueocean.rest.model.BlueScmConfig;
-import jenkins.branch.OrganizationFolder;
-import jenkins.model.Jenkins;
-import jenkins.scm.api.SCMNavigator;
-import org.acegisecurity.Authentication;
-import org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator;
-import org.kohsuke.stapler.DataBoundConstructor;
-
-import javax.annotation.Nonnull;
-import java.io.IOException;
-import java.util.List;
-
-/**
- * @author Vivek Pandey
- */
-public class GithubPipelineUpdateRequest extends BluePipelineUpdateRequest {
```

Comment:
I think It's not used to ok to remove but will let @kzantow confirm as it was left out there for some use case, probably something in future.